.github/workflows/check.yml

```diff
@@ -107,4 +107,4 @@ jobs:
         run: |
           SKIP_WASM_BUILD=1 cargo test
           SKIP_WASM_BUILD=1 cargo clean -p picasso-runtime
-          SKIP_WASM_BUILD=1 cargo clean -p composable
\ No newline at end of file
+          SKIP_WASM_BUILD=1 cargo clean -p composable
```

Comment:
By the way do we still need the SKIP_WASM_BUILD ???

Reply:
imho, can remove `SKIP_WASM_BUILD=1` before `cargo clean ...`.